lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runime/azbatch/driver/AzureBatchDriverConfiguration.java

```diff
@@ -28,7 +32,13 @@
 import org.apache.reef.runtime.common.launch.parameters.ErrorHandlerRID;
 import org.apache.reef.runtime.common.launch.parameters.LaunchID;
 import org.apache.reef.runtime.common.parameters.JVMHeapSlack;
-import org.apache.reef.tang.formats.*;
+import org.apache.reef.runtime.hdinsight.parameters.AzureStorageAccountContainerName;
```

Comment:
FYI: on Friday I checked in a change that adds these parameters to azure batch runtime. I also completely removed hdinisght runtime dependency from batch runtime. So you may want to merge the changes.

Reply:
I will rebase it before merging the change.

Reply:
done